decisions/wc_united_sacrifice_decision.txt

```diff
@@ -0,0 +1,60 @@
+targetted_decisions = {
+	united_old_gods_sacrifice_prisoner_decision = {
+		only_playable = yes
+		filter = court
+		ai_target_filter = court
+		ai_check_interval = 12
+
+		diplomacy_icon = GFX_old_gods_sacrifice_decision
+
+		from_potential = { can_sacrifice_characters_to_og_potential_trigger = yes }
+
+		potential = {
+			prisoner = yes
+			can_be_sacrificed_trigger = yes
+			is_untouchable_trigger = no
+			host = { character = FROM }
+		}
+
+		allow = {
+			NOT = { diplomatic_immunity = yes }
+			FROM = { can_sacrifice_characters_to_og_allow_trigger = yes }
+		}
+
+		effect = {
+			FROM = {
+				show_scope_change = no
+
+				ROOT = {
+					show_scope_change = no
+
+					caltulate_PREV_offmap_old_gods_currency_insanity_effect = yes
```

Comment:
Why it's used outside `PREV_sacrifice_THIS_to_old_god_effect`? It's always used together.

Reply:
Good catch